governance/cost-check.sh

```
@@ -29,6 +29,23 @@ ok()    { echo "OK    $1"; }
 act()   { echo "ACT   $1"; }
 infra() { echo "INFRA $1" >&2; INFRA=$((INFRA+1)); }
 
+# ---------- AUDIT（ADR-0057，INV-12：宪法 §11 行 3 预算检查的审计条目） ----------
+# 本脚本纳入管家唤醒矩阵（cron 6h→1h）。trigger 由 workflow 注入 COST_TRIGGER
+# （${{ github.event_name }}：schedule/workflow_dispatch——"谁唤醒"）；头行=running，
+# 尾行由 EXIT 陷阱按实际退出码落（0=ok 1=tripped 2=infra-fail）——多出口脚本无需
+# 逐出口插行，判定逻辑零改动。duration 由 butler-audit.sh 的审计起点口径计算
+# （source 时刻起算，等效脚本内 SECONDS）。
+source "$DIR/butler-audit.sh" || { echo "FATAL: butler-audit.sh 加载失败" >&2; exit 2; }
```

Shellcheck (0.11.0)

[info] Not following: ./butler-audit.sh was not specified as input (see shellcheck -x).

(SC1091)